internal/configs/node/group.go

```diff
@@ -25,32 +21,50 @@ func (id GroupID) String() string {
 
 // Const of group ID
 const (
-	GroupIDBeacon            GroupID = "harmony/0.0.1/node/beacon"
-	GroupIDBeaconClient      GroupID = "harmony/0.0.1/client/beacon"
-	GroupIDShardPrefix       GroupID = "harmony/0.0.1/node/shard/%s"
-	GroupIDShardClientPrefix GroupID = "harmony/0.0.1/client/shard/%s"
-	GroupIDGlobal            GroupID = "harmony/0.0.1/node/global"
-	GroupIDGlobalClient      GroupID = "harmony/0.0.1/node/global"
-	GroupIDUnknown           GroupID = "B1acKh0lE"
+	GroupIDBeacon            GroupID = "%s/0.0.1/node/beacon"
+	GroupIDBeaconClient      GroupID = "%s/0.0.1/client/beacon"
+	GroupIDShardPrefix       GroupID = "%s/0.0.1/node/shard/%s"
+	GroupIDShardClientPrefix GroupID = "%s/0.0.1/client/shard/%s"
+	GroupIDGlobal            GroupID = "%s/0.0.1/node/global"
+	GroupIDGlobalClient      GroupID = "%s/0.0.1/node/global"
+	GroupIDUnknown           GroupID = "%s/B1acKh0lE"
 )
 
 // ShardID defines the ID of a shard
 type ShardID uint32
 
+func getNetworkPrefix(shardID ShardID) (netPre string) {
+	switch GetShardConfig(uint32(shardID)).GetNetworkType() {
+	case Mainnet:
+		netPre = "harmony"
```

Comment:
Let's do "harmony-mainnet", "harmony-testnet", "harmony-pangaea" ....  to avoid potential conflict with other projects.

Reply:
Just realize that it's related to backward compatibility, so should be "harmony", "harmony-testnet", "harmony-pangaea"

Reply:
sure will change it.